Changes

```diff
@@ -126,6 +126,9 @@ Working version
   min_max_num to module Float.
   (Christophe Troestler, review by Alain Frish, Xavier Clerc and Daniel Bünzli)
 
+- GPR#1354: Add fma support to Float module.
+  (Laurent Thévenoux)
```

Comment:
Please include people involved in the review.

Reply:
Reviewers only or all participants?

Reply:
Looking quickly at the discussion, I'd say : @jhjourdan @xavierleroy @alainfrisch

Reply:
Thanks, done!